Source/Tools/LinuxEmulation/VDSO_Emulation.cpp

```diff
@@ -595,8 +664,8 @@ uint64_t GetVSyscallEntry(const void* VDSOBase) {
   return 0;
 }
 
-const std::span<FEXCore::IR::ThunkDefinition> GetVDSOThunkDefinitions() {
-  return VDSODefinitions;
+const std::span<FEXCore::IR::ThunkDefinition> GetVDSOThunkDefinitions(bool Is64Bit) {
+  return std::span(VDSODefinitions.begin(), VDSODefinitions.begin() + VDSODefinitions.size() - (Is64Bit ? 0 : 1));
```

Comment:
Why not `end()`?

```suggestion
  return std::span(VDSODefinitions.begin(), VDSODefinitions.end() - Is64Bit);
```

This could also be useful:
```suggestion
  return std::span { VDSODefinitions }.first(VDSODefinitions.size() - Is64Bit);
```

Reply:
Went with the first example since semantically I don't like the looks of the second.